### PR TITLE
feat: home and product list page

### PR DESCRIPTION
**Pull request by tewarimohit**
https://user-images.githubusercontent.com/89827782/153625446-ec301479-7f74-476d-94d6-209c06d11112.mp4

### CODE REVIEW

**Comment by spraveenofficial** [comment]
Code looks good, one thing you also deploy so that we can check the UI too.

**Review by spraveenofficial — APPROVED** [review]

File: css_docs/home.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/* :root{
+    margin: 0 ;
+padding: 0 ;
+} */
```
Comment (abhijeetekad) [inline css_docs/home.css:4]: You can remove this.

File: css_docs/products.css [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    
+  }
+  
+  .page-content {
+    grid-area: content;
+   display: grid;
+grid-template-columns: repeat(4,1fr);
+gap:5px;
+align-items: flex-start;
+}
```
Comment (abhijeetekad) [inline css_docs/products.css:54]: Look once for indentation. use prettier for this if you are not using it already.

**Review by abhijeetekad — APPROVED** [review]
Added some suggestions..look for it. otherwise website looks great.

**Pull request merged**